cluster/clean.sh

```diff
@@ -12,9 +12,10 @@ if [ ! -d $MANIFESTS_DIR ]; then
 fi
 
 $kubectl delete --ignore-not-found -f $MANIFESTS_DIR/operator.yaml
-$kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nodenetworkconfigurationenactments_crd.yaml
-$kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nodenetworkconfigurationpolicies_crd.yaml
-$kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nodenetworkstates_crd.yaml
+$kubectl delete --ignore-not-found -f deploy/crds/nmstate.io_nodenetworkconfigurationenactments_crd.yaml
+$kubectl delete --ignore-not-found -f deploy/crds/nmstate.io_nodenetworkconfigurationpolicies_crd.yaml
+$kubectl delete --ignore-not-found -f deploy/crds/nmstate.io_nodenetworkstates_crd.yaml
```

Comment:
Isn't NMState CR the owner of the operand CRDs ?

Reply:
@qinqon , that make sense that NMState will be the owner of operand CRDs,  do you expect that in 'real' cluster, deletion of NMState instance means deletions for handler CRDs?
If your answer is yes, I guess NMState controller should be updated.
an another Q, do we assume that NMState is singletone ? (max number of  NMState instances = 1) ?